examples/react/css/src/common.css

```diff
@@ -0,0 +1,61 @@
+/* Motion reduction */
+@media (prefers-reduced-motion: reduce) {
+  [class^='ais-'] {
+    transition: none !important;
+  }
+}
```

Comment:
should this live in the `reset`?

Reply:
No strong opinion on this one, WDYT @sarahdayan?

Reply:
Can be argued, but this ruleset is about disabling opinionated styles in themes, not about resetting browser defaults, so common sounds fine.